core/src/services/azblob/core.rs

```diff
@@ -533,6 +643,13 @@ impl AzblobCore {
     }
 }
 
+/// Request of CompleteMultipartUploadRequest
+#[derive(Default, Debug, Serialize)]
+#[serde(default, rename = "<BlockList>", rename_all = "PascalCase")]
```

Comment:
I believe the rename here is wrong. We will need an extra wrapper here for block list struct.